storybook/storybook-react/src/components/RadioButton/RadioButton.stories.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { RadioButton } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/radio-button/radio-button.scss'
+
+const meta = {
+  title: 'Radio Button',
+  component: RadioButton,
+  args: {
+    checked: false,
+    invalid: false,
+    disabled: false,
+    onChange: () => {
+      console.log('onChange')
+    },
```

Comment:
I don't know it this is very useful. Currently, it logs an entry when you click on it, but the state of the radio button doesn't actually change